wormhole-connect/src/config/utils.ts

```diff
@@ -157,65 +157,65 @@ export const validateDefaults = (
   tokens: TokensConfig,
 ) => {
   if (!defaults) return;
-  if (defaults.fromNetwork) {
-    const chain = chains[defaults.fromNetwork];
+  if (defaults.fromChain) {
+    const chain = chains[defaults.fromChain];
     if (!chain) {
       error(
-        `Invalid chain name "${defaults.fromNetwork}" specified for bridgeDefaults.fromNetwork`,
+        `Invalid chain name "${defaults.fromChain}" specified for bridgeDefaults.fromChain`,
       );
-      delete defaults.fromNetwork;
+      delete defaults.fromChain;
     }
   }
-  if (defaults.toNetwork) {
-    const chain = chains[defaults.toNetwork];
+  if (defaults.toChain) {
+    const chain = chains[defaults.toChain];
     if (!chain) {
       error(
-        `Invalid chain name "${defaults.toNetwork}" specified for bridgeDefaults.toNetwork`,
+        `Invalid chain name "${defaults.toChain}" specified for bridgeDefaults.toChain`,
       );
-      delete defaults.fromNetwork;
+      delete defaults.fromChain;
     }
   }
-  if (defaults.fromNetwork && defaults.toNetwork) {
-    if (defaults.fromNetwork === defaults.toNetwork) {
+  if (defaults.fromChain && defaults.toChain) {
+    if (defaults.fromChain === defaults.toChain) {
       error(
         `Source and destination chain cannot be the same, check the bridgeDefaults configuration`,
       );
     }
   }
 
-  if (defaults.fromNetwork && defaults.toNetwork && defaults.requiredNetwork) {
+  if (defaults.fromChain && defaults.toChain && defaults.requiredNetwork) {
     const requiredConfig = chains[defaults.requiredNetwork];
     if (!requiredConfig) {
       error(
         `Invalid network value "${defaults.requiredNetwork}" specified for bridgeDefaults.requiredNetwork`,
       );
     }
     if (
-      defaults.toNetwork !== defaults.requiredNetwork &&
-      defaults.fromNetwork !== defaults.requiredNetwork
+      defaults.toChain !== defaults.requiredNetwork &&
```

Comment:
Should we rename `requiredNetwork` to `requiredChain`?

Reply:
ah... good catch thanks. will do